tsl/src/continuous_aggs/create.c

```diff
@@ -1055,6 +1055,19 @@ cagg_query_supported(Query *query, StringInfo hint, StringInfo detail, const boo
 		return false;
 	}
 
+#if PG15_GE
+	if (!finalized)
```

Comment:
I guess this validation should be the first one in the `cagg_query_supported` function, no?